### DISCUSSION TITLE
Repository creation hangs

### DISCUSSION THREAD

**Discussion by vamshidharkibm** [community/community#25157] [Actions]
<p>Hi,</p>
<p>The below command from the shell script is working fine on my Red Hat Enterprise Linux.</p>
<p>The control hangs at this line when it is executed from a shell script from the gitbash on a windows system.</p>
<p>What could be the issue?</p>
<p><strong>curl -u $gitUserName <a href="https://api.github.com/user/repos" rel="nofollow noopener">https://api.github.com/user/repos</a> -d “{“name”:”$repoName"}"</strong></p>
<p>Thanks</p>
<p>Vamshi</p>

**Comment by vamshidharkibm** [comment, marked as answer]
<p>It works now with the following</p>
<p><strong>winpty curl</strong></p>
<p>Thanks</p>
<p>Vamshi</p>

**Comment by Nguyenduongtruongtoan2020** [comment]
<h1>This is a basic workflow to help you get started with Actions</h1>
<p>name: CI</p>
<h1>Controls when the action will run. Triggers the workflow on push or pull request</h1>
<h1>events but only for the master branch</h1>
<p>on:<br>
push:<br>
branches: [master]<br>
pull_request:<br>
branches: [master]</p>
<h1>A workflow run is made up of one or more jobs that can run sequentially or in parallel</h1>
<p>jobs:</p>
<h1>This workflow contains a single job called “build”</h1>
<p>build:</p>
<h1>The type of runner that the job will run on</h1>
<p>runs-on: ubuntu-latest</p>
<h1>Steps represent a sequence of tasks that will be executed as part of the job</h1>
<p>steps:</p>
<h1>Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it</h1>
<ul>
<li>uses: actions/checkout@v2</li>
</ul>
<h1>Runs a single command using the runners shell</h1>
<ul>
<li>name: Run a one-line script<br>
run: echo Hello, world!</li>
</ul>
<h1>Runs a set of commands using the runners shell</h1>
<ul>
<li>name: Run a multi-line script<br>
run: |<br>
echo Add other actions to build,<br>
echo test, and deploy your project.</li>
</ul>
